content/cli/v7/commands/npm.md

```diff
@@ -40,7 +40,7 @@ Run `npm help` to get a list of available commands.
 npm comes preconfigured to use npm's public registry at
 https://registry.npmjs.org by default. Use of the npm public registry is
 subject to terms of use available at
-https://www.npmjs.com/policies/terms.
+https://docs.npmjs.com/policies/terms.
```

Comment:
This needs to be updated in https://github.com/npm/cli/blob/latest/docs/content/commands/npm.md